src/Model/Behavior/DuplicatableBehavior.php

```diff
@@ -91,13 +95,13 @@ public function duplicateEntity(int|string $id): EntityInterface
 
         foreach ($this->getConfig('remove') as $field) {
             $parts = explode('.', $field);
-            $this->_drillDownEntity('remove', $entity, $parts);
+            $this->_drillDownEntity('remove', $entity, $parts, null, $original);
```

Comment:
Don't like having to explicitly pass `null` for the `$value` arg when it's not needed. We can add the arg `$original` after `$entity` instead.

Reply:
The main issue was here to keep it in sync for both set and remove
Since they have different argument order
But directly after $entity seems fine probably.

Reply:
Done, also added a test.